swap_meet/clothing.py

```diff
@@ -1,2 +1,9 @@
-class Clothing:
-    pass
\ No newline at end of file
+from .item import Item
+
+class Clothing(Item):
+    def __init__(self, category="Clothing", condition=0.0):
+        self.category = category
+        self.condition = condition
```

Comment:
This works but I think we can make this more efficient!

Given that we repeat L5-6 in the parent class Item's initializer, we can simplify this by using the super initializer:
```
def __init__(self, condition=0.0): 
    super().__init__(category="Clothing", condition=condition)
```